<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1192</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1181</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Crud</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>301</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox{

color:rgb(24, 50, 109);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
  



    border: 3px solid white;



}</string>
      </property>
      <property name="title">
       <string>Les Ev√®nements</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font: 63 10pt &quot;Segoe UI Semibold&quot;;
color : rgb(24, 50, 109);
}</string>
       </property>
       <property name="text">
        <string>Date De  Debut</string>
       </property>
      </widget>
      <widget class="QLabel" name="df">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font: 63 10pt &quot;Segoe UI Semibold&quot;;
color : rgb(24, 50, 109);
}</string>
       </property>
       <property name="text">
        <string>Date de fin</string>
       </property>
      </widget>
      <widget class="QLabel" name="lieu">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>190</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font: 63 10pt &quot;Segoe UI Semibold&quot;;
color : rgb(24, 50, 109);
}</string>
       </property>
       <property name="text">
        <string>lieu</string>
       </property>
      </widget>
      <widget class="QLabel" name="nb_p">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font: 63 10pt &quot;Segoe UI Semibold&quot;;
color : rgb(24, 50, 109);
}</string>
       </property>
       <property name="text">
        <string>Nb Participants</string>
       </property>
      </widget>
      <widget class="QLabel" name="budget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>260</y>
         <width>56</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font: 63 10pt &quot;Segoe UI Semibold&quot;;
color : rgb(24, 50, 109);
}</string>
       </property>
       <property name="text">
        <string>Budget:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>330</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 50%;
background-color:rgb(24, 50, 109);
color:white;
 border-width: 0px;
  border-radius: 10px;
}

</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_ref">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nb_p">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>230</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_modify">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>330</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 50%;
background-color:rgb(24, 50, 109);
color:white;
 border-width: 0px;
  border-radius: 10px;
}

</string>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QLabel" name="reference">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font: 63 10pt &quot;Segoe UI Semibold&quot;;
color : rgb(24, 50, 109);
}</string>
       </property>
       <property name="text">
        <string>Reference</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_budget">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>270</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QDateEdit" name="le_dd">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>110</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="le_df">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>150</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="le_lieu">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>190</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Tunis</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sfax</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sousse</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QLineEdit" name="le_chercher">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>20</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	font: 63 10pt &quot;Segoe UI Semibold&quot;;
 

color:rgb(24, 50, 109);
}
</string>
      </property>
      <property name="text">
       <string>Tri par :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>1070</x>
        <y>540</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 50%;
background-color:rgb(24, 50, 109);
color:white;
 border-width: 0px;
  border-radius: 10px;
}
</string>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supp_event">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>510</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 50%;
background-color:rgb(24, 50, 109);
color:white;
 border-width: 0px;
  border-radius: 10px;
}

</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="supp_ref">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>510</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lieu_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>510</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
font: 63 10pt &quot;Segoe UI Semibold&quot;;
color : rgb(24, 50, 109);
}</string>
      </property>
      <property name="text">
       <string>REF</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	font: 63 10pt &quot;Segoe UI Semibold&quot;;
 

color:rgb(24, 50, 109);
}
</string>
      </property>
      <property name="text">
       <string>Rechercher:</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_events_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>180</y>
        <width>781</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>10</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>REF</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>30</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>LIEU</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>50</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>NB_P</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Stats">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>540</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 50%;
background-color:rgb(24, 50, 109);
color:white;
 border-width: 0px;
  border-radius: 10px;
}
</string>
      </property>
      <property name="text">
       <string>Stats</string>
      </property>
     </widget>
     <zorder>groupBox</zorder>
     <zorder>label_7</zorder>
     <zorder>le_chercher</zorder>
     <zorder>pushButton_4</zorder>
     <zorder>pb_supp_event</zorder>
     <zorder>supp_ref</zorder>
     <zorder>lieu_2</zorder>
     <zorder>label_8</zorder>
     <zorder>tab_events_2</zorder>
     <zorder>radioButton</zorder>
     <zorder>radioButton_2</zorder>
     <zorder>radioButton_3</zorder>
     <zorder>Stats</zorder>
    </widget>
    <widget class="QWidget" name="Mailing">
     <attribute name="title">
      <string>Mailing</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>691</width>
        <height>471</height>
       </rect>
      </property>
      <property name="title">
       <string>Mailing</string>
      </property>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>371</width>
         <height>451</height>
        </rect>
       </property>
       <property name="title">
        <string>Mailing</string>
       </property>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Email :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_13">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>30</y>
          <width>171</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Objet</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_14">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>90</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Message</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>380</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Envoyer</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="msgmail">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>190</y>
          <width>281</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Calendrier">
     <attribute name="title">
      <string>Calendrier</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>1051</width>
        <height>511</height>
       </rect>
      </property>
      <property name="title">
       <string>Calendrier</string>
      </property>
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>851</width>
         <height>218</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableView" name="tab_events">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>270</y>
         <width>781</width>
         <height>211</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1192</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
